runelite-client/src/main/java/net/runelite/client/game/FishingSpot.java

```diff
@@ -196,13 +197,19 @@
 	}
 
 	FishingSpot(String spot, int fishSpriteId, int... ids)
+	{
+		this(spot, spot, fishSpriteId, ids);
+	}
+
+	FishingSpot(String spot, String worldMapTooltip, int fishSpriteId, int... ids)
 	{
 		this.name = spot;
+		this.worldMapTooltip = worldMapTooltip;
```

Comment:
The `worldMapTooltip` addition should be added in the same commit as FishingSpotLocation and the other world map code is added, not the one where `FishingSpot` is moved to a different package.

Reply:
fixed the commits and rebased